src/picker.common.ts

```diff
@@ -96,31 +96,34 @@ export class PickerField extends TextField implements TemplatedItemsView {
 
         this.applyCssScope(this._page.actionBar);
 
-        let actionItem = new ActionItem();
-        actionItem.text = "Close";
-        actionItem.on(Button.tapEvent, (args: ItemEventData) => {
-            this.closeCallback(undefined, undefined);
-        });
+        const isNavigationButton = this.androidCloseButtonPosition === "navigationButton";
```

Comment:
Will this preserve the previous behaviour on iOS as could cause the created instance on iOS to be `NavigationButton ` due to `androidCloseButtonPosition` being set to `navigationButton`